decideMain.html

```diff
@@ -4,10 +4,10 @@
     <title>decideMain</title>
     <!-- <link rel="stylesheet" href="mainStyle.css"> -->
     <script type='text/javascript'
-            src='decideMain.js'></script>
+            src='decideMainJS'></script>
   </head>
   <body>
-    <form id="groupForm" action="decideQuestion" method="get">
+    <form id="groupForm" onsubmit="makeNewURL()" method="post">
```

Comment:
This looks like its a post where other stuff is get requests?

Reply:
Crud. I had updated this, but hadn't pushed to the remote. I'll push now. It will have a different attempt (using axios I think)